src/go/rdctl/cmd/shutdown.go

```diff
@@ -31,15 +48,144 @@ var shutdownCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		result, err := processRequestForUtility(doRequest("PUT", versionCommand("", "shutdown")))
+		cmd.SilenceUsage = true
+		result, err := doShutdown()
 		if err != nil {
 			return err
 		}
-		fmt.Println(string(result))
+		if result != nil {
+			fmt.Println(string(result))
+		}
 		return nil
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(shutdownCmd)
+	shutdownCmd.Flags().BoolVar(&shutdownSettings.Verbose, "verbose", false, "be verbose")
+	shutdownCmd.Flags().BoolVar(&shutdownSettings.DontWait, "dont-wait", false, "don't wait for shutdown to be confirmed")
+}
+
+func doShutdown() ([]byte, error) {
+	output, err := processRequestForUtility(doRequest("PUT", versionCommand("", "shutdown")))
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
There's also a chance that the config file exists, but the server isn't running.

Reply:
In which case they either get an error message explaining that the server isn't running, or they get the text of some other unanticipated error.